ios/CameraView+Focus.swift

```diff
@@ -74,24 +74,64 @@ extension CameraView {
       }
 
       // in {0..1} system
-      let normalizedPoint = captureDevicePointConverted(fromLayerPoint: point)
+      var normalizedPoint: CGPoint
+      if let previewView = previewView as? PreviewView {
+        // previewView is of type PreviewView can use the built in captureDevicePointConverted
+        normalizedPoint = previewView.videoPreviewLayer.captureDevicePointConverted(fromLayerPoint: point)
+      } else {
+        normalizedPoint = captureDevicePointConverted(fromLayerPoint: point)
+      }
```

Comment:
Let's remove this separate check and just make sure the `captureDevicePointConverted` method works as expected. That way we are future proof - maybe there could be a different Preview View type in the future.

Reply:
I am of a different opinion here. I think it is better to use the built in Apple method to do this point conversion when available. Also at the moment `captureDevicePointConverted` does not work. If you log the output of `previewView.videoPreviewLayer.captureDevicePointConverted(fromLayerPoint: point)` and `captureDevicePointConverted(fromLayerPoint: point)` they produce completely different results. 

If there is a different Preview View type in the future the code would not compile and would be easy enough to resolve at that time.

Can a TODO be added as a task to resolve the discrepancies between the custom method and the built in one? I honestly do not have time at the moment to dig into this any more but would like to get this fix merged so that it is fixed for when people are not using the skia preview layer.